pipeline: output: vivo exporter: style (#1987)
fluent/fluent-bit-docs: pipeline/outputs/vivo-exporter.md

|------------|-------------------------------------------------------------------------------------------------------------------------------|
| `/logs`    | Exposes log events in JSON format. Each event contains a timestamp, metadata and the event content.                           |
| `/metrics` | Exposes metrics events in JSON format. Each metric contains name, metadata, metric type and labels (dimensions).              |
| `/traces`  | Exposes traces events in JSON format. Each trace contains a name, resource spans, spans, attributes, events information, and so on. |

The following example generates dummy log events which will be consuming by using `curl` HTTP command line client:

1. Configure and start Fluent Bit

   {% tabs %}
   {% tab title="fluent-bit.yaml" %}

   ```yaml
   pipeline:
     inputs:
       - name: dummy
         tag: events
         rate: 2

     outputs:
       - name: vivo_exporter
         match: '*'
   ```

   {% endtab %}
   {% tab title="fluent-bit.conf" %}

   ```text
   [INPUT]
     name  dummy
     tag   events
     rate  2

   [OUTPUT]
     name   vivo_exporter
     match  *
   ```

   {% endtab %}
   {% endtabs %}

> 1. Retrieve the data

Reviewer: ```suggestion
1. Retrieve the data.
```

### Streams and IDs

> As mentioned previously, each stream buffers a `chunk` that contains `N` events, each chunk contains its own ID which is unique inside the stream.

Reviewer: ```suggestion
As mentioned previously, each stream buffers a `chunk` that contains `N` events, with each chunk containing its own ID that's unique inside the stream.
```

The values of these headers can be used by the client application to specify a range between IDs or set limits for the number of chunks to retrieve from the stream.

### Retrieve ranges and use limits

A client might be interested in always retrieving the latest chunks available and skip previous ones already processed. In a first request without any given range, Vivo Exporter will provide all the content that exists in the buffer for the specific stream. On that response, the client might want to keep the last ID (`Vivo-Stream-End-ID`) that was received.

To query ranges or starting from specific chunks IDs, remember that they're incremental. You can use a mix of the following options:

| Query string option | Description |
|---------------------|-------------|
| `from` | Specify the first chunk ID to be retrieved. If the `chunk` ID doesn't exist, the next one in the queue will be provided. |
| `to` | The last chunk ID to be retrieved. If not found, the whole stream will be provided (starting from `from` if was set). |
| `limit` | Limit the output to a specific number of chunks. The default value is `0`, which sends everything. |

> The following example specifies the range from chunk ID `1` to chunk ID `3` and only 1 chunk:

Reviewer: ```suggestion
The following example specifies the range from chunk ID `1` to chunk ID `3` and only one chunk:
```
